src/test/regress/base_schedule

```diff
@@ -1,8 +1,8 @@
 # ----------
 # Only run few basic tests to set up a testing environment
 # ----------
+test: multi_test_helpers multi_test_helpers_superuser multi_create_fdw columnar_test_helpers failure_test_helpers
```

Comment:
What about mini_schedule? Do we need to update that as well?

Reply:
I guess that makes sense too, we can take care of that in another pr